doc_indexer/src/fetcher/fetcher.py

```diff
@@ -52,11 +52,9 @@ def fetch_documents(self, source: DocumentsSource) -> None:
         try:
             scroller = Scroller(repo_dir, module_output_dir, source)
             scroller.scroll()
-        except Exception as e:
-            logger.error(
-                f"Error while scrolling documents for: {source.name}: {str(e)}"
-            )
-            raise e
+        except Exception:
+            logger.exception(f"Error while scrolling documents for: {source.name}")
+            raise
```

Comment:
Will `raise` implicitly raise the actual Exception?

Reply:
Also, shall we document somewhere these common recommendations to avoid future security issues?

Reply:
yes, it re-raises the current exception.

I've just refactored it and the error is still logged in the error log. It is normal to log the error that we need in the logs. No end user can see it.

The issue was with the error messages added to messages that I removed.

Reply:
shall we document somewhere these common recommendations to avoid future security issues?

Reply:
this is just refactoring that is not related to security issue.